packages/opds-web-client/src/AuthPlugin.ts

```diff
@@ -1,14 +1,26 @@
-import { AuthFormProps, AuthButtonProps } from "./components/AuthProviderSelectionForm";
-import { AuthMethod, AuthCredentials } from "./interfaces";
+import {
+  AuthFormProps,
+  AuthButtonProps
+} from "./components/AuthProviderSelectionForm";
+import {AuthMethod, AuthCredentials} from "./interfaces";
 
 /** Applications can implement this interface if they would like to support authentication
     methods other than basic auth. A list of `AuthPlugin`s can be passed as a prop to the
     `OPDSCatalog` component. */
 interface AuthPlugin {
   type: string;
-  lookForCredentials: () => { credentials?: AuthCredentials; error?: string; } | void;
-  formComponent: new(props: AuthFormProps<AuthMethod>) => React.Component<AuthFormProps<AuthMethod>, {}>;
-  buttonComponent: new(props: AuthButtonProps<AuthMethod>) => React.Component<AuthButtonProps<AuthMethod>, {}>;
+  lookForCredentials: () => {
+    credentials?: AuthCredentials;
+    error?: string;
+  } | void;
+  formComponent: new (props: AuthFormProps<AuthMethod>) => React.Component<
+    AuthFormProps<AuthMethod>,
+    {}
+  >;
+  buttonComponent: new (props: AuthButtonProps<AuthMethod>) => React.Component<
+    AuthButtonProps<AuthMethod>,
+    {}
```

Comment:
Are these being broken up because it's a long line or because each param should be on its own line?

Reply:
Because it's a long line. They try to keep lines below their `printWidth` setting, which is currently 80. We can set it to longer if you like. There are exceptions where they won't break the line up, like a string literal for example.